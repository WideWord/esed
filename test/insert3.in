ewfmew f
wefwef
few fwefwe
ew fwef ef
w efewfwef
