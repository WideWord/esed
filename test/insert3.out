ewfmew f
ghghgh
wefwef
few fwefwe
ew fwef ef
w efewfwef
